chart/templates/rendered.yaml

```diff
@@ -544,7 +813,7 @@ spec:
         env:
         - name: FLAGD_VERSION
           value: v0.2.5
-        image: ghcr.io/open-feature/open-feature-operator:main
+        image: controller:latest
```

Comment:
Can you explain why this change is being made?

Reply:
This was unintentional, reverting